python/lsst/ts/fbs/utils/maintel/roman_surveys.py

```diff
@@ -214,22 +232,40 @@ def gen_roman_off_season(
     camera_ddf_rot_per_visit : `float`
         The rotational dither offset to apply per visit.
         In degrees.
-    exptime : `float`
-        The exposure time per visit.
-    nexp : `int`
-        The number of snaps (exposures) per visit.
+    sequence : `str`
+        Band names to use in the sequence
+    nvisits : `dict`
+        Number of visits for each band. Passed to FieldSurvey.
+    nexps : `dict`
+        Number of exposures for each band. Passed to FieldSurvey.
+    exptimes : `dict`
+        Exposure times for each band. Passed to FieldSurvey.
     science_program : `str`
         Name of the science program for the survey.
     safety_mask_params : `dict`
         A dictionary of additional kwargs to pass to the standard safety masks.
 
     Returns
     -------
-    survey : `DeepDrillingSurvey`
+    survey : `FieldSurvey`
         A survey configured to observe in a sequence of 'griz'
         every third day while the RGES field is visible but not being
         observed by Roman.
     """
+    # updated these to 2 visits before filter change
+    # original plan is 1 per band -- can AOS handle this?
```

Comment:
Is this comment a note for future work to return to 1 per band after some analysis or just a note? If it's not ticketed, can we remove the comment?

Reply:
It's not ticketed, but it is a note for me. This is removed in the new version which just passes the value to the FieldSurvey though.